packages/expo-document-picker/ios/EXDocumentPicker/EXDocumentPickerModule.m

```diff
@@ -78,20 +86,18 @@ @implementation EXDocumentPicker
   
   // Because of the way IPad works with Actionsheets such as this one, we need to provide a source view and set it's position.
   if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) {
-    documentMenuVC.popoverPresentationController.sourceRect = CGRectMake(CGRectGetMidX([RCTPresentedViewController().view frame]), CGRectGetMaxY([RCTPresentedViewController().view frame]), 0, 0);
-    documentMenuVC.popoverPresentationController.sourceView = RCTPresentedViewController().view;
+    documentMenuVC.popoverPresentationController.sourceRect = CGRectMake(CGRectGetMidX([_utilities.currentViewController.view frame]), CGRectGetMaxY([_utilities.currentViewController.view frame]), 0, 0);
```

Comment:
`RCTPresentedViewController` ➡️ `_utilities.currentViewController` substitution isn't always valid, have you tried out this code? (just checking)

Reply:
Yeap, I've tried it, moreover I've checked what's happening in `_utilities.currentViewController` and it's almost exact the same as in `RCTPresentedViewController`.